src/contracts/fund/hub/Hub.ts

```diff
@@ -164,7 +193,25 @@ export class Hub extends Contract {
    * @param from The sender address.
    */
   public setRouting(from: Address) {
-    return this.createTransaction('setRouting', from);
+    const validate = async () => {
+      await this.validateCreator(from);
+
+      let message;
```

Comment:
This should not be a let but instead use a const in the respective if() block. That i spossible because every `{ }` has its own context.